lib/pure/concurrency/cpuinfo.nim

```diff
@@ -49,6 +49,9 @@ when defined(haiku):
 proc countProcessors*(): int {.rtl, extern: "ncpi$1".} =
   ## returns the number of the processors/cores the machine has.
   ## Returns 0 if it cannot be detected.
+  ##
+  ## .. code-block:: nim
```

Comment:
`runnableExamples:`?

Reply:
Honestly I dont know the difference between `cpuinfo.countProcessors()` and `osproc.countProcessors()` then I used the `code-block` way.
Both work Ok tho. :man_shrugging:

Reply:
one is implemented in terms of the other:
```nim
proc countProcessors*(): int {.rtl, extern: "nosp$1".} =
  ## Returns the number of the processors/cores the machine has.
  ## Returns 0 if it cannot be detected.
  result = cpuinfo.countProcessors()
```
and in fact (to avoid your confusion) the docs in osproc should mention that it's just calling `cpuinfo.countProcessors`